assets/src/dashboard/components/cardGridItem/cardTitle.js

```diff
@@ -137,7 +137,7 @@ const CardTitle = ({
           {titleFormatted(title)}
         </TitleStoryLink>
       )}
-      <TitleBodyText>
+      <TitleBodyText className={'dashboard-item-date'}>
```

Comment:
```suggestion
      <TitleBodyText className="dashboard-item-date">
```

Could also use `dashboard-grid-item-date` to make it clearer.